Remotion/Web/UnitTests/Core/ExecutionEngine/Infrastructure/WxePageStepExecutionStates/Execute/ExecutingSubFunctionWithoutPermaUrlStateTest.cs

```diff
@@ -49,31 +50,29 @@ public void IsExecuting ()
     [Test]
     public void ExecuteSubFunction_GoesToPostProcessingSubFunction ()
     {
-      using (MockRepository.Ordered())
-      {
-        SubFunction.Expect (mock => mock.Execute (WxeContext));
-        ExecutionStateContextMock.Expect (mock => mock.SetExecutionState (Arg<PostProcessingSubFunctionState>.Is.NotNull))
-            .WhenCalled (invocation => CheckExecutionState ((PostProcessingSubFunctionState) invocation.Arguments[0]));
-      }
-
-      MockRepository.ReplayAll();
+      var sequence = new MockSequence();
+      SubFunction.InSequence (sequence).Setup (mock => mock.Execute (WxeContext)).Verifiable();
+      ExecutionStateContextMock.InSequence (sequence)
+          .Setup (mock => mock.SetExecutionState (It.IsNotNull<PostProcessingSubFunctionState>()))
+          .Callback ((IExecutionState executionState) => CheckExecutionState ((PostProcessingSubFunctionState) executionState))
+          .Verifiable();
 
       _executionState.ExecuteSubFunction (WxeContext);
 
-      MockRepository.VerifyAll();
+      ExecutionStateContextMock.Verify();
```

Comment:
We should consider adding a VerifyAll Method to the base class and using that one here (since the verifies got added manually anyway and there's a bunch of them in several files.